client/extensions/woocommerce/app/store-stats/store-stats-chart/index.js

```diff
@@ -111,7 +112,11 @@ class StoreStatsChart extends Component {
 									selected={ tabIndex === selectedTabIndex }
 									tabClick={ this.tabClick }
 								>
-									<span className="store-stats-chart__value value">{ itemChartData.value }</span>
+									<span className="store-stats-chart__value value">
+										{ ( tab.type === 'currency' )
+											? formatCurrency( itemChartData.value )
```

Comment:
Would you be able to link me to an example in the shipping code? I had presumed that `formatCurrency()` was aware of the current store currency settings.